_docs/ios/1.141.0.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: 1.141.0
+permalink: /docs/ios/1.141.0/
+parent: iOS
+nav_order: -141
+---
+
+## Wi-Fi settings checking for Auto-unlock is stored for the same user on different mobile devices.
+Wi-Fi settings for checking for Auto-unlock is now stored altogether with Auto-unlock settings. That means, that this settings will be shared between all mobile devices that are log in in to same user.
+
+## The new settings page for Auto-unlock zones with a map.
+Auto-unlock zones settings get redesign and now it has new screen with map. Zones are visualized on the map for better understanding how they work and where Auto-unlock process will be started (by entering range in zone) and where Auto-unlock process will be stopped (by leaving range out zone).
```

Comment:
"...how they work, where Auto-unlock process will be started...", use comma instead of "and"